src/tests/ftest/process_core_files.py

```diff
@@ -48,11 +52,15 @@ def process_core_files(self, directory, delete):
                 process
             delete (bool): delete the core files.
 
+        Raises:
+            CoreFileException: if there is an error processing core files.
+
         Returns:
-            int: number of errors encountered
+            int: num of corefile processesd
```

Comment:
To alleviate both pylint
```suggestion
            int: num of core files processed
```